subprojects/docs/src/docs/release/notes.md

```diff
@@ -63,6 +63,26 @@ users of the plugin and so should be used carefully.
 
 For more details see (user manual link)
 
+
+## Improvements for "ComponentMetadataRule"s
+
+The API for component metadata rules has been extended to allow access to `packaging`, `classifier` and `type` information from POM metadata.
+The `packaging` information for a direct dependency [can be obtained](userguide/component_metadata_rules.html#filter_using_maven_metadata) from the `ComponentMetadataContext`.
+Additionally, `ComponentMetadataRule`s can react on the `classifier` and `type` information of the transitive dependencies:
```

Comment:
```suggestion
Additionally, `ComponentMetadataRule`s can react on the `classifier` and `type` information of dependencies:
```

I _think_ this is better as from the point of view of the rule, they are just 'the dependencies'.